plugin/sampling/strategystore/static/strategy_store_test.go

```diff
@@ -465,18 +467,44 @@ func TestAutoUpdateStrategyErrors(t *testing.T) {
 }
 
 func TestServiceNoPerOperationStrategies(t *testing.T) {
+	// given setup of strategy store with no specific per operation sampling strategies
 	store, err := NewStrategyStore(Options{StrategiesFile: "fixtures/service_no_per_operation.json"}, zap.NewNop())
 	require.NoError(t, err)
 
-	s, err := store.GetSamplingStrategy(context.Background(), "ServiceA")
+	// expected response for ServiceA (which has probablistic service level sampling strategy)
+	// shall contain /health operation with default service level sampling strategy as well
+	expectedServiceAResponse, errServiceA := makeServiceStrategyResponseFromFile("fixtures/TestServiceNoPerOperationStrategies_response_for_ServiceA.json")
```

Comment:
nit: one of the useful techniques I used when dealing with fixtures for __expected__ outcomes (aka "snapshots") is to have an env var that allows the test to re-generate the snapshots (e.g. cmd/query/app/apiv3/gateway_test.go). Since you have relatively small outputs I don't think that specific mechanism is necessary, but the way snapshots are generated is by using the test name as the file name: https://github.com/jaegertracing/jaeger/blob/55ceba9467114d3bbbfad9c4bdfa27b96c28b1f5/cmd/query/app/apiv3/gateway_test.go#L80

You did it partially, but you still have a manually added suffix `response_for_ServiceA`. You can use just the test name if you create sub-tests within this function, e.g. 
```
t.Run("serviceA", func(t) {
  expectedFile := basePath + t.Name()
});
```

Reply:
Sounds like a way to be lazy about updating JSON's... I love it :D